src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -340,6 +334,29 @@ private TagBuilder GenerateTextBox(ModelExplorer modelExplorer, string inputType
                 htmlAttributes: htmlAttributes);
         }
 
+        private TagBuilder GenerateHidden(ModelExplorer modelExplorer)
```

Comment:
Definitely deserves a comment about mimicking `IHtmlGenerator.GenerateHidden()` with added `format` support. In particular, mention call to that protected `GenerateInput()` method in the `DefaultHtmlGenerator` treats `InputType.Text` and `InputType.Hidden` identically if the given HTML attributes already contain `type`.